app/main.py

```diff
@@ -2,0 +1,12 @@
+
+def copy_file(command: str) -> None:
+    date = command.split()
+    if date[0] != "cp":
+        return
+    first_txt, second_txt = date[1], date[2]
+    if first_txt == second_txt:
```

Comment:
This check is good for preventing the user from copying a file to itself. However, it might be better to raise an error or print a message to the user when the source and destination are the same, rather than silently returning.